PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Uncomment source, md5sums, cd to add in AUR.
+pkgname="snuffleupagus"
+pkgver=r142.73c30e3
+pkgrel=1
+pkgdesc="Snuffleupagus module for PHP7."
+url="https://snuffleupagus.readthedocs.io"
+arch=('i686' 'x86_64')
```

Comment:
Because Arch is fun, this is the [official]( https://wiki.archlinux.org/index.php/PKGBUILD#arch ) way to specify that the package builds on any architecture.